Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -54,41 +62,44 @@ final class PledgeCTAContainerView: UIView {
   override func bindStyles() {
     super.bindStyles()
 
-    _ = self.pledgeCTAbutton
-      |> projectStateButtonStyle
+    let isAccessibilityCategory = self.traitCollection.preferredContentSizeCategory.isAccessibilityCategory
 
-    _ = self.youreABackerLabel
-      |> \.font .~ .ksr_headline(size: 14)
-      |> \.text %~ { _ in Strings.Youre_a_backer() }
+    _ = self.amountAndRewardTitleStackView
+      |> \.axis .~ NSLayoutConstraint.Axis.vertical
+      |> \.isLayoutMarginsRelativeArrangement .~ true
 
     _ = self.amountOrRewardLabel
-      |> \.font .~ .ksr_caption1(size: 14)
+      |> \.font .~ .ksr_headline(size: 14)
       |> \.textColor .~ .ksr_dark_grey_500
+      |> \.numberOfLines .~ 0
```

Comment:
| Before 🐛 | After 🦋 |
| --- | --- |
| <img width="460" alt="Screen Shot 2019-06-25 at 5 49 33 PM copy" src="https://user-images.githubusercontent.com/387596/60143750-a31b8100-9774-11e9-840d-8a8f9b19356f.png"> | <img width="460" alt="Screen Shot 2019-06-25 at 5 50 21 PM copy" src="https://user-images.githubusercontent.com/387596/60143758-a878cb80-9774-11e9-9951-6bfa70a14f95.png"> |

Reply:
I'm wondering if we should to handle Dynamic Type on fixed elements differently. The viewable part of the screen (beneath the `PledgeCTAContainerView`) is getting pretty tight!